### DISCUSSION TITLE
Investigate Citus schema based sharding

### DISCUSSION THREAD

**Discussion by stevefan1999-personal** [FerretDB/FerretDB#3094] [General]
https://www.citusdata.com/blog/2023/07/18/citus-12-schema-based-sharding-for-postgres/

It seems like we have a way to automatically shard FerretDB thanks to this (although having ObjectId does optimize further!)

**Comment by AlekSi** [comment]
Thank you for letting us know. We are going to investigate that in #226 or related issues.
